scripts/bytecodecompare/storebytecode.sh

```diff
@@ -82,6 +82,12 @@ EOF
 
     if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]
     then
+        if !(test -f report.txt)
```

Comment:
Are you sure this is the error? I was assuming that travis already created exactly the same report on exactly this commit in the past and that is why `git commit` shows "no change".